### PR TITLE
Update symbols

### PR DESCRIPTION
**Pull request by groverlynn**
- Fix a bug where `<` is omitted from unicodes
- Correct wrong unicodes
- Delete duplicates and put paired symbols together

### CODE REVIEW

**Comment by sunderme** [comment]
can you describe "bug where < is omitted from unicodes" ?

**Comment by groverlynn** [comment]
> can you describe "bug where < is omitted from unicodes" ?

@sunderme  When the unicode character(s) for a symbol contains `<` (here, only several instances with the literal "<", but could also apply to "<⃥" or "<⃒"), inserting the unicode from the symbol list gives wrong character (empty). `<` should be escaped as `&lt;` to get the correct unicode character(s).

**Comment by sunderme** [comment]
please state an exact test case, where the current behavior leads to unwanted behavior.
Your change fails when using "insert as unicode" on "<".
grafik

**Comment by sunderme** [comment]
I would suggest that you take out the change on src/symbolpanel/symbollistmodel.cpp.
Especially as that is something different from the rest.

**Comment by groverlynn** [comment]
Ok... escaped `<` too early. Now it works.

Screenshot 2023-01-26 at 02 17 17 (2)

**Comment by sunderme** [comment]
toHtmlEscaped() seems to be the simpler solution.

File: src/symbolpanel/symbollistmodel.cpp [hunk 1]
```diff
@@ -86,10 +86,10 @@ SymbolItem loadSymbolFromSvg(QString fileName)
 	if (!nl.isEmpty()) {
 		QDomNode n = nl.at(0);
 		QString pkg = n.toElement().attribute("Packages");
-		if (!pkg.isEmpty())
+		if (!pkg.isEmpty() && pkg != "")
 			item.packages = pkg;
 		QString cmdUnicode = n.toElement().attribute("CommandUnicode");
-		if (!cmdUnicode.isEmpty())
+		if (!cmdUnicode.isEmpty() && cmdUnicode!="")
```
Comment (sunderme) [inline src/symbolpanel/symbollistmodel.cpp:92]: `&& cmdUnicode!=""`is redundadnt, please remove

File: src/symbolpanel/symbollistmodel.cpp [hunk 2]
```diff
@@ -86,10 +86,10 @@ SymbolItem loadSymbolFromSvg(QString fileName)
 	if (!nl.isEmpty()) {
 		QDomNode n = nl.at(0);
 		QString pkg = n.toElement().attribute("Packages");
-		if (!pkg.isEmpty())
+		if (!pkg.isEmpty() && pkg != "")
```
Comment (sunderme) [inline src/symbolpanel/symbollistmodel.cpp:89]: `&& pkg!=""`is redundant, please remove

**Comment by sunderme** [comment]
thanks

**Pull request merged**